components/usage-api/usage/v1/usage.proto

```diff
@@ -34,10 +34,18 @@ message ListBilledUsageRequest {
     }
 
     Ordering order = 4;
+
+    int64 per_page = 5;
+    int64 page = 6;
 }
 
 message ListBilledUsageResponse {
     repeated BilledSession sessions = 1;
+    int64 per_page = 2;
+    int64 total_pages = 3;
+    int64 total = 4;
+    int64 page = 5;
```

Comment:
This should also be encapsulated in a Pagination object

Reply:
👍🏻 thanks for the hint to encapsulate the pagination concerns.